### DISCUSSION TITLE
Replaceable pieces of content

### DISCUSSION THREAD

**Discussion by square-foot** [GitbookIO/community#795] [Feature Requests]
Hi
Not sure if this has already been implemented and that I am merely unaware of it.  

I sorely miss an easy way to change some critical values in the ongoing documentation. For e.g. I may want to change the pricing of my product (for which I am writing the docs)  Instead of searching and replacing the information regarding pricing everytime I change it, there could be some "value placeholder" and a separate configuration containing "name-value" pairs. So if I write (as an example) <:pricing:> in various parts of the book,  then before publishing Gitbook could search for the key "pricing" from the configuration where the name-value pairs are listed, and use the value stated for that key for the final published version. That way I can change the critical value only at one place.

This would make a nice product even greater. Thanks for reading till here.

**Comment by sregier206** [comment]
100% need this. In ServiceNow, this is called knowledge blocks. In RoboHelp I believe it is called a Snippet. In RoboHelp, you can use it for big blocks of text, or even things as small as trademarked names in case they change. 

This would greatly reduce linking and ensure a SSOT (single source of truth). 

It would be nice to have user access lockdown on this as well, but allow all user to see the list of these available for their use.
